epub33/overview/index.html

```diff
@@ -697,10 +690,10 @@ <h3>EPUB 3.1: 2017</h3>
 				<h3>EPUB 3.2: 2018</h3>
 				<p>The work on EPUB 3.2 was undertaken shortly after EPUB 3.1 to restore compatibility of content to
 					EPUB 3. The change of version number introduced in EPUB 3.1 meant that EPUB Creators, vendors and
-					reading system developers would have to produce, distribute and consume two versions of EPUB
+					Reading System developers would have to produce, distribute and consume two versions of EPUB
 					content, but the costs of this change outweighed the benefits of the new version. EPUB 3.2 instead
-					keeps all the best parts of EPUB 3.1 but returns and deprecates the removed elements so that a new
-					version number is not necessary in the Package Document. [[EPUB-32]] [[EPUBPackages-32]]
+					keeps all the best parts of EPUB 3.1 but deprecates the removed elements so that a new
```

Comment:
Maybe just me, but it sounds now like the elements are both removed and deprecated.
```suggestion
					keeps all the best parts of EPUB 3.1 but deprecates elements instead of removing them so that a new
```